src/pages/CourseContent/index.js

```diff
@@ -49,8 +50,23 @@ function CourseContent(props) {
   const courseId = get(props, "match.params.courseId");
 
   useEffect(() => {
-    dispatch(courseActions.getCourseContent({ courseId: courseId }));
-  }, [dispatch, courseId]);
+    const getLocalStorageValue = localStorage.getItem("changeLanguage");
+    const valueSet =
+      getLocalStorageValue === undefined
+        ? setChangeLanguage("hi")
```

Comment:
Considering we cannot guarantee that every course will have Hindi content by default let's keep `en` as default option for now.

Reply:
Done